compat/test/browser/suspense.test.js

```diff
@@ -382,11 +382,7 @@ describe('suspense', () => {
 			expect(scratch.innerHTML).to.eql(`<p>Count: 2</p>`);
 			expect(componentWillMount).to.have.been.calledOnce;
 			expect(componentDidMount).to.have.been.calledOnce;
-			// TODO: This is called thrice since the cDU queued up after the second
-			// increment is never cleared once the component suspends. So when it
-			// resumes and the component is rerendered, we queue up another cDU so
-			// cDU is called an extra time.
-			expect(componentDidUpdate).to.have.been.calledThrice;
```

Comment:
Since `_renderCallbacks` is stored on VNode now, they don't persist across renders. If a VNode suspends, it'll never get added to the `commitQueue` array so its `_renderCallbacks` won't get called. And since VNodes are recreated on each render it won't get invoked on the next render since that'll be with a newVNode, and not the one from the previous render which had the `_renderCallback`.